flasksrc/session.py

```diff
@@ -86,6 +86,15 @@ class ToyNetSessionByIdCreateRouterPutResp(Schema):
     pass
 
 
+class ToyNetSessionByIdCreateRouterInterfacePutReq(Schema):
+    name = fields.Str()
+    ip = fields.Str()
```

Comment:
Can you comment examples at least for the ip?

Reply:
I added a comment; since doing so makes this class inconsistent with the rest of the schema classes I think we should have a follow-up issue to inline document the rest of them across the codebase.

Reply:
I see.
Personally, I think commenting is useful something like `name` where a specific format is not required. I specifically would like to add a comment in cases where user might want some quick help knowing how to get a sample http request working. What do you think?